### DISCUSSION TITLE
Error Notice - You can't perform that action at this time error

### DISCUSSION THREAD

**Discussion by anantajitjg** [community/community#8640] [Other Feature Feedback, Questions, & Ideas] [answered]
When simply clicking **_View lists_** link I am getting this error notice and also gets redirected to the Lists page.

![Clicking View Lists](https://user-images.githubusercontent.com/22009263/145523722-989a4f85-8577-495e-8004-2deb846ff026.png)

![Error Notice](https://user-images.githubusercontent.com/22009263/145523766-13874050-acc7-4666-81ef-39db5a474b2d.png)

**Comment by SWxEng** [comment]
I also have the same problem.......how do you solve?

**Reply by SWxEng** [reply]
??????????

**Comment by jfullstackdev** [comment]
try logging in in Incognito mode and see whether it's the fix

**Reply by SWxEng** [reply]
does not work

**Reply by SWxEng** [reply]
Kongklai ???

**Comment by anantajitjg** [comment, marked as answer]
Currently, the UI is different. The issue at that time was resolved after clearing the web browser's cache and logging in again to the account.

<img width="740" alt="Screenshot 2022-09-16 at 8 47 40 AM" src="https://user-images.githubusercontent.com/22009263/190549965-100abfb6-c47f-4012-ab4c-14c60cf1edd4.png">

**Comment by Vmrbee03** [comment]
Unstall and reinstall

**Reply by Vmrbee03** [reply]
I love my kids and would never do anything to hurt them intentionally.Im hurting without them.

**Reply by SWxEng** [reply]
ok

**Reply by JackXammie** [reply]
go get them then or ask for my help, its important you get now that you are certain.

**Comment by Lumitex** [comment]
.

**Reply by Lumitex** [reply]
..

**Reply by SWxEng** [reply]
????

**Comment by nepalauziamas** [comment]
Sss

**Reply by SWxEng** [reply]
????????
